pkg/interface/chat/src/js/components/lib/header-bar.js

```diff
@@ -7,7 +7,7 @@ export class HeaderBar extends Component {
   render() {
     let spin = (this.props.spinner)
       ?  <div className="absolute"
-           style={{width: 16, height: 16, top: 16, right: 16}}>
+           style={{width: 16, height: 16, top: 16, left: 55}}>
```

Comment:
I feel weird using an odd number given the pattern here. Is 64px what we really want?

Reply:
That would look like this btw

![image](https://user-images.githubusercontent.com/20846414/62813376-657e7780-bad8-11e9-978d-704c74fdd765.png)